src/components/main/index.tsx

```diff
@@ -1,21 +1,23 @@
 import React from 'react';
+import { useRouter } from 'next/router';
 
 import HomeLayout from '../common/layout/layout';
 import Beer from '../common/beer/beer';
 import PreferenesModal from './preferenes-pop-up';
-
 import { MainContainer, MainContent, MainTab, Text, TabButton } from './styled';
 
-import Data, { DummyProps } from './dummy';
+import { useRatesBeer } from '../../api/hook/beers';
+import { IBeer } from '../../api/types/beers';
 
 const Main = () => {
+  const router = useRouter();
+
   const [activeTab, setActiveTab] = React.useState<string>('0');
+  const { data: ratesData } = useRatesBeer();
 
-  const tabMenuHandle = (name: string): void => {
-    // if(name === "preferences" && !performance){
-    // TODO : 모달 띄워주기
-    // }
-    setActiveTab((prev: string) => name);
+  const tabMenuHandle = (tabNum: string): void => {
+    // TODO : 유저 선호맥주 선택 여부 모달 띄워주기
+    setActiveTab(() => tabNum);
```

Comment:
activeTab을 숫자로 하지 않고 문자열로 해도 되지 않을까요?

```typescript
const homeTabs = {
RATES : "RATES",
PREFERENCE :"PREFERENCE"
};

Object.freeze(homeTabs);

```
```suggestion
  const [activeTab, setActiveTab] = React.useState<string>(homeTabs.RATES);
  const { data: ratesData } = useRatesBeer();
  const tabMenuHandle = (tabName: string): void => {
    // TODO : 유저 선호맥주 선택 여부 모달 띄워주기
    setActiveTab(tabName);
    };
```

Reply:
비비말대로 이렇게 표현하는게 더 명확한거같아요!
저도 안그래도 뭔가 찜찜했는데 이렇게 수정하겠습니다 ^0^ !

Reply:
비비말대로 이렇게 표현하는게 더 명확한거같아요!
styled 파일까지 같이 수정해서 푸쉬하겠습니답